src/blocks/cpu.rs

```diff
@@ -7,6 +7,9 @@
 //! `format` | A string to customise the output of this block. See below for available placeholders. | `" $icon $utilization "`
 //! `format_alt` | If set, block will switch between `format` and `format_alt` on every click | `None`
 //! `interval` | Update interval in seconds | `5`
+//! `info_cpu` | Percentage of CPU usage, where state is set to info | `0.3`
```

Comment:
I'd write that the defaults are 30.0, 60.0, 90.0

Reply:
Oh sorry, forgot to change that
I'll quickly fix this